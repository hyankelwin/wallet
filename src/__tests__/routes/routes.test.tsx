import { shallow } from 'enzyme';

import { BrowserRouter as Router } from 'react-router-dom';
import Routes from '../../routes';
import RouteProps from '../../routes/Route';

describe('Routes page', () => {
  it('should render Routes with success', () => {
    const wrapper = shallow(<Routes />);

    expect(wrapper.debug()).toMatchSnapshot();
  });

  it('should render RouteProps with success', () => {
    const wrapper = shallow(
      <Router>
        <RouteProps />
      </Router>,
    );
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
