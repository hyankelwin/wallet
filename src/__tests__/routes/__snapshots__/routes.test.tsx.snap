// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes page should render RouteProps with success 1`] = `
"<Router history={{...}}>
  <Route />
</Router>"
`;

exports[`Routes page should render Routes with success 1`] = `
"<Switch>
  <Route path=\\"/\\" exact={true} component={[Function: Home]} />
</Switch>"
`;
